### DISCUSSION TITLE
My education pack is approved but still i am not able to redeem benefits

### DISCUSSION THREAD

**Discussion by Kuba0734** [community/community#118434] [GitHub Education]
### Hi everyone,

My application is approved but still i am not able to access anything.
![iddmage](https://github.com/community/community/assets/126418851/129fd510-c1e7-4168-833d-7d792cd492a8)

**Comment by balusena** [comment]
If your education pack is approved but you're unable to redeem benefits, ensure you've completed all necessary steps for activation. Double-check your eligibility and any required verification processes. Contact GitHub Education support for assistance, providing your application ID and relevant details for expedited resolution.

**Comment by samus-aran** [comment]
Hi @Kuba0734  👋 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
